test/gtest/ucp/test_ucp_fence.cc

```diff
@@ -154,6 +154,10 @@ class test_ucp_fence : public test_ucp_atomic {
         params.length     = memheap_size;
         params.flags      = GetParam().variant;
 
+        if (params.flags & UCP_MEM_MAP_FIXED) {
```

Comment:
Because the case is wrong when the memory was allocated by user then this address is using for fixed mapping. mmap returns success but the pages will cause a segfault after unmap